### DISCUSSION TITLE
Opening page from background

### DISCUSSION THREAD

**Discussion by mrklos** [zepp-health/zeppos-forum#394] [Q&A] [answered]
Is there a possibility to open page (for example same as in Counter system app after counting has ended) from app service?
I am aware that you can use notification, but they can be ignored/missed easly. If it's not possible. Can we request it in future? :)

**Comment by MyoungXUE** [comment, marked as answer]
Directly open the Mini Program page is disturbing to users and is not recommended, so there is a notification function.

But you can try using the set method of the @zos/alarm module in App Service, which may achieve your idea.

https://docs.zepp.com/docs/reference/device-app-api/newAPI/alarm/set

**Reply by JulianBessone** [reply]
Dear MyoungXUE,

I'm trying to create an Alarm in my App Service, but I can't, can you say me if there is a problem with my code?

`
const alarmTime = Date.now() + 10000; // now + 10 seg

	const option = {
		url: 'page/index.js',
		time: alarmTime,
	}
	console.log(option.url, option.time)
	const id = set(option)
	console.log('The result of creating alarm is: ', id)
`

**Reply by mrklos** [reply]
Make sure you include page/index in app.json like this:
"page": {
          "pages": [
            "page/index"
          ]
        },

and make sure to have 
  "permissions": [
    "device:os.alarm",
  ],

**Reply by JulianBessone** [reply]
Dear @mrklos ,

The resolution for my problem was that the **set()** method for the @alarm must have the time property expressed in **seconds**. I was using time expressed in milliseconds. I missed that detail in de Zepp OS doc. I'm very ashamed of having done that. Thanks for your all support.
